syncopy/datatype/base_data.py

```diff
@@ -207,7 +207,7 @@ def _register_dataset(self, propertyName, inData=None):
 
         supportedSetters[type(inData)](inData, propertyName, ndim=ndim)
 
-    def _unregister_dataset(self, propertyName, del_from_file=True):
+    def _unregister_dataset(self, propertyName, del_from_file=True, del_attr=True):
```

Comment:
yup, makes sense here! Real dynamic properties would be the most flexible solution here, let's see maybe we still manage that..